packages/mask/package.json

```diff
@@ -116,6 +119,7 @@
     "socket.io-client": "2.4.0",
     "stream-browserify": "^3.0.0",
     "stream-http": "^2.8.3",
+    "ulid": "^2.3.0",
```

Comment:
Reason?

```js
crypto.randomUUID()
```

Reply:
Yes, changed crypto.randomUUID, didn’t know that it has good browser coverage.